torch/_inductor/utils.py

```diff
@@ -117,8 +116,8 @@ def timed(model, example_inputs, times=1):
 
 
 def print_performance(fn, args=(), times=10, repeat=10, baseline=1.0):
-    timings = [timed(fn, args, times) for _ in range(repeat)]
-    took = np.median(timings)
+    timings = torch.tensor([timed(fn, args, times) for _ in range(repeat)])
+    took = torch.median(timings)
```

Comment:
just curious, whether it introduces problems with printing below? e.g. can a scalar tensor be formatted directly with `f"{scalar_tensor:.6f}"` or does it need to say `f"{float(scalar_tensor):.6f}"` to convert a scalar tensor to a float?

Reply:
Seems that the scalar tensor can be formatted just fine.